### PR TITLE
Add authentication-related templates

### PR DESCRIPTION
**Pull request by josegonzalez**
- [x] Login action template
- [x] Login view template
- [x] Logout action template

### CODE REVIEW

**Comment by josegonzalez** [comment]
Anyone have any thoughts on this? They are pretty basic, but essentially pulled straight from the book auth tutorial.

File: src/Template/Bake/Element/Controller/login.ctp [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+%>
+
+    /**
+     * Login method
+     *
+     * @return void
+     */
+    public function login() {
```
Comment (antograssiot) [inline src/Template/Bake/Element/Controller/login.ctp]: opening braket on new line maybe to please the PSR2 :trollface:

**Comment by lorenzo** [comment]
How do you bake those only for one controller?

**Comment by josegonzalez** [comment]
Pretty sure this line already *only* sets the actions to the existing ones, and will properly exclude these two.

https://github.com/cakephp/bake/blob/master/src/Shell/Task/ControllerTask.php#L94

**Comment by lorenzo** [comment]
@josegonzalez ok, so how do you bake the controller code then?

**Comment by markstory** [comment]
@lorenzo We could add a `--actions` CLI option to list out the specific actions people want included. That could also be used to address #164

**Comment by lorenzo** [comment]
@markstory that would make sense

**Comment by markstory** [comment]
@josegonzalez Do you want to add the `--actions` flag? If not I can take care of it.

**Comment by simkimsia** [comment]
:+1: for auth related bake templates

**Comment by josegonzalez** [comment]
@markstory I'd rather you did since I'll be missing the test :(

**Comment by markstory** [comment]
@josegonzalez Cool. I will get to it this week then.

**Comment by josegonzalez** [comment]
@lorenzo atm you can just set them via a bake event.

**Comment by markstory** [comment]
I've merged these templates as #198 adds the CLI flag to whitelist actions into a controller.

**Pull request merged**